src/components/loading/AposSpinner.vue

```diff
@@ -1,27 +1,27 @@
 <template>
   <div class="apos-spinner">
-    <svg 
-      xmlns="http://www.w3.org/2000/svg" 
-      class="apos-spinner__svg" 
-      width="38" 
-      height="38" 
+    <svg
+      xmlns="http://www.w3.org/2000/svg"
```

Comment:
They can have more than one per line. But the second line of attributes are expected to line up with the first. So rather than deep indent the second (in some cases) I'm bumping down the first line to be two spaces in.